data-prepper-plugins/mutate-string-processors/README.md

```diff
@@ -0,0 +1,196 @@
+# Mutate String Processors
```

Comment:
I'm not going to block the PR because of this, but I am going to advocate for what I consider to be a cleaner format for the READMEs that provides links to each of the configuration options so that are easy to navigate similar to the README for the Aggregate Processor. 

For this README, I suggest links like below:


```
# Mutate String Processors

* [substitute_string]()
* [split_string]()
* [uppercase_string]()
* .... remaining string processors

```

Since these configuration descriptions are very short, it's probably not necessary to link to each of the options are the start of the README for each processor.

Reply:
Although each Processor section was short, I decided to take your suggestion and add links at the top so the user can only go to a specified section. We can do this for other multiple processors in Data Prepper too. Thanks for the feedback.